### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by safi-alt** [RobinLinus/snapdrop#155]
Cannot use btween Windows 10 and Andriod 8.1
Tried using after installing PWA too.

**Comment by erayrafet** [comment]
Same issue on Android 10 and Windows 10 with Chrome and the legacy Edge on the PC.

**Comment by amiayan** [comment]
Snapdrop not working between my Nokia 8.1, A10 and Windows 10 laptop , I use Edge chromium

**Comment by kubark42** [comment]
I'm also having issues. The setup is macOS 10.14.5 and Windows 10. From macOS we tried to send with both Chrome and Safari. Interestingly, I could send between browsers on the same computer, but we couldn't send files or messages between computers.

We also tried installing the Chrome app, but no joy.

There is no indication of success or failure, we see each other but there is no feedback in the UI about the transfer state. It seems to silently fail.
